dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/liberty/CDIRuntimeImpl.java

```diff
@@ -114,6 +116,9 @@
     @Reference(name = "beanParser", service = BeanParser.class)
     private BeanParser beanParser;
```

Comment:
Remind me why I called this a "BeanParser"? It should have been "BeansXMLParser" :-(